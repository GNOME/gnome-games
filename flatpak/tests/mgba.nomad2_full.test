[Retro Reftest]
Path=/mgba/nomad2_full
Core=/app/lib/libretro/mgba_libretro.so
Medias=nomad2_full.gba;

[Frame 120]
Video=mgba.nomad2_full.120.png
