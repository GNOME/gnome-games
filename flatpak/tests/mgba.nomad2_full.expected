/mgba/nomad2_full/Boot: OK
/mgba/nomad2_full/120/FastForward: OK
/mgba/nomad2_full/120/Run: OK
/mgba/nomad2_full/120/Video: OK
